exasol/toolbox/nox/tasks.py

```diff
@@ -239,6 +245,38 @@ def _coverage(
     session.run(*command)
 
 
+@nox.session(name="audit", python=False)
+def audit(session: Session) -> None:
+    """Audit project dependencies in regard of their license."""
+
+    def _packages():
+        result = run(
+            ["poetry", "show", "--no-ansi", "--no-interaction", "--only", "main"],
+            capture_output=True,
+            check=True,
+        )
+        stream = TextIOWrapper(BytesIO(result.stdout))
+        lines = (line for line in stream)
+        return (line.split(" ")[0] for line in lines)
+
+    packages = [
+        package
+        for package in _packages()
+        if package not in PROJECT_CONFIG.audit_exceptions
+    ]
+    session.run(
+        "poetry",
+        "run",
+        "python",
+        "-m",
+        "piplicenses",
+        "--packages",
+        *packages,
```

Comment:
in that case you must hope, that you don't have more packages as the command line allows parameters.https://www.cyberciti.biz/faq/linux-unix-arg_max-maximum-length-of-arguments/

Reply:
https://learn.microsoft.com/en-us/troubleshoot/windows-client/shell-experience/command-line-string-limitation